src/utils/attributeUtils.tsx

```diff
@@ -0,0 +1,37 @@
+export const shortenPgpOrReturnEmail = (url: string) => {
+  if (url === "") return;
+
+  const regex = new RegExp(
+    /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)/
```

Comment:
add a comment describing what this is lol

Reply:
👍